src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -14,130 +15,138 @@ import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrap
-  const goToNextPage = () => navigate(config.routes.SSV.VALIDATOR.DISTRIBUTION_METHOD.UPLOAD_KEYSHARES);
-  const {
-    operatorsIds,
-    operatorsKeys,
-  } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr, operator: IOperator) => {
-    // @ts-ignore
-    aggr.operatorsIds.push(operator.id);
-    // @ts-ignore
-    aggr.operatorsKeys.push(operator.public_key);
-    return aggr;
-  }, {
-    operatorsIds: [],
-    operatorsKeys: [],
-  });
-  const cliCommand = `--operator-keys=${operatorsKeys.join(',')} --operator-ids=${operatorsIds.join(',')}`;
+    const instructions = [
+        {
+            id: 1, instructions: [
+                <Grid>1. Download the <b>MacOS</b> executable from <LinkText text={'SSV-Keys Github'}
+                                                                             link={'https://github.com/bloxapp/ssv-keys/releases'}/></Grid>,
+                '2. Launch your terminal',
+                '3. Navigate to the directory you downladed the CLI tool',
+                '4. Run the tool with the following command:',
+            ],
+        },
+        {
+            id: 2, instructions: [
+                '1. Download the MacOS app from  the Starkeys Github',
+                '2.Run the Starkeys app',
+                '3. When prompted, copy and paste the following command:',
+            ],
+        },
+    ];
 
-  const instructions = [
-    {
-      id: 1, instructions: [
-        <Grid>1. Download the <b>MacOS</b> executable from <LinkText text={'SSV-Keys Github'}
-                                                                     link={'https://github.com/bloxapp/ssv-keys/releases'}/></Grid>,
-        '2. Launch your terminal',
-        '3. Navigate to the directory you downladed the CLI tool',
-        '4. Run the tool with the following command:',
-      ],
-    },
-    {
-      id: 2, instructions: [
-        '1. Download the MacOS app from  the Starkeys Github',
-        '2.Run the Starkeys app',
-        '3. When prompted, copy and paste the following command:',
-      ],
-    },
-  ];
+    const copyToClipboard = () => {
+        navigator.clipboard.writeText(cliCommand);
+        notificationsStore.showMessage('Copied to clipboard.', 'success');
+        setTextCopied(true);
+    };
 
-  const copyToClipboard = () => {
-    navigator.clipboard.writeText(cliCommand);
-    notificationsStore.showMessage('Copied to clipboard.', 'success');
-    setTextCopied(true);
-  };
+    const checkBox = (id: number) => {
+        setTextCopied(false);
+        setSelectedBox(id);
+    };
 
-  const checkBox = (id: number) => {
-    setTextCopied(false);
-    setSelectedBox(id);
-  };
+    const copyButton = () => {
+        if (!textCopied) return <Grid item className={classes.CopyButton} onClick={copyToClipboard}>Copy</Grid>;
+        return <Grid onClick={copyToClipboard} container item className={classes.ButtonCopied}>
```

Comment:
Extract copy button to standalone component with properties passing to it

Reply:
done